apps/emqx_rule_engine/src/emqx_rule_engine_api.erl

```diff
@@ -419,6 +444,16 @@ param_path_id() ->
             {404, #{code => 'NOT_FOUND', message => <<"Rule Id Not Found">>}}
     end.
 
+'/rule_engine'(get, _Params) ->
+    {200, format_rule_engine_resp(emqx_conf:get([rule_engine]))};
```

Comment:
Tiny nit: would it be better to encapsulate this `emqx_conf` call and the update below inside `emqx_rule_engine`'s API?  Something like `emqx_rule_engine:get_configs/0`?

Reply:
The suggestion was to maybe wrap the `emqx_conf` call in an API inside `emqx_rule_engine` for encapsulation.  Like:

```erlang
%% emqx_rule_engine.erl
get_configs() ->
    emqx_conf:get([rule_engine]).
```